english/cricket-rules/cricket-rule-book.md

```diff
@@ -0,0 +1,6 @@
+Rules and game-play
+
+Cricket is a bat and ball game, played between two teams of eleven players each. 
+One team bats, attempting to score runs, while the other bowls and fields the ball, attempting to restrict the scoring and dismiss the batsmen. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.